Yosemite/Yosemite/Stores/SitePluginStore.swift

```diff
@@ -100,6 +115,22 @@ private extension SitePluginStore {
             }
         }
     }
+
+    func isPluginActive(siteID: Int64, plugin: SitePluginAction.Plugin, completion: @escaping (Result<Bool, Error>) -> Void) {
+        Task { @MainActor in
+            do {
+                let plugins = try await remote.loadPluginsFromWPCOM(siteID: siteID)
+                let isJetpackActive = plugins.contains(where: { $0.id == plugin.id && $0.isActive })
+                await MainActor.run {
+                    completion(.success(isJetpackActive))
+                }
+            } catch {
+                await MainActor.run {
+                    completion(.failure(error))
+                }
```

Comment:
Good question, I was suggested to call the completion block in a previous code review to be safe because `@MainActor` in a `Task` used to be not very reliable. I just tested with a breakpoint with and without calling the completion in `await MainActor.run`, and the completion block was called on the main thread in both cases. I removed this `await MainActor.run` in https://github.com/woocommerce/woocommerce-ios/pull/9731/commits/9dcbdeb1fc15afeb40178eb6f42aff5262ceb321 to simplify the code.